File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/DataGridView/PopulatingData/BindingDataSourcePage.cs
#region Using

using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using Gizmox.WebGUI.Common;
using Gizmox.WebGUI.Forms;

#endregion

namespace CompanionKit.Controls.DataGridView.PopulatingData
{
    public partial class BindingDataSourcePage : UserControl
    {
        public BindingDataSourcePage()
        {
            InitializeComponent();
            //Allow grouping
            this.mobjDataGridView.ShowGroupingDropArea = true;
        }        

        private void BindingDataSourcePage_Load(object sender, EventArgs e)
        {
            //Fill up table adapter with data from table
            this.mobjCustomersTableAdapter.Fill(this.mobjNorthwindDataSet.Customers);
        }
    }
}

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/DataGridView/PopulatingData/BindingDataSourcePage.vb
Namespace CompanionKit.Controls.DataGridView.PopulatingData

	Public Class BindingDataSourcePage

		Public Sub New()

			' This call is required by the Visual WebGui UserControl Designer.
            InitializeComponent()
            'Allow grouping
            Me.mobjDataGridView.ShowGroupingDropArea = True

		End Sub

        Private Sub BindingDataSourcePage_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
            ' Fill up table adapter with data from table
            Me.mobjCustomersTableAdapter.Fill(Me.mobjNorthwindDataSet.Customers)
        End Sub
	End Class

End Namespace